compiler-core/src/parse/error.rs

```diff
@@ -372,6 +373,20 @@ impl LexicalError {
             LexicalErrorType::InvalidUnicodeEscape(InvalidUnicodeEscapeError::InvalidCodepoint) => {
                 ("Invalid Unicode codepoint", vec![])
             }
+            LexicalErrorType::InvalidOperator { expected } if *expected == "==" => (
+                "Did you mean `==`?",
+                vec![
+                    "Gleam uses `==` to check for equality between two values.".into(),
+                    "See: https://tour.gleam.run/basics/equality".into(),
+                ],
+            ),
+            LexicalErrorType::InvalidOperator { expected } => (
+                "This is not a valid operator at this context",
+                vec![
+                    "Hint: Is it a typo?".into(),
+                    format!("Try: `{}` operator instead.", expected),
+                ],
+            ),
```

Comment:
I'd personally define a specific `InvalidTripleEqual` instead of a general one since we don't currently have any uses for it besides the triple equal

Reply:
Thank you for the review updated the commit 👍🏽

Reply:
No, thank you!